modules/authentication/proc-enabling-authentication-with-microsoft-azure.adoc

```diff
@@ -0,0 +1,130 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-adding-azure-as-an-authentication-provider_{context}"]
+= Enabling the Microsoft Azure authentication provider
+
+{product} includes a Microsoft Azure authentication provider that can authenticate users by using OAuth.
+
+.Prerequisites
+. You have the permission to register an application in Microsoft Azure.
+. You link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/administration_guide_for_red_hat_developer_hub/index#assembly-add-custom-app-file-openshift_admin-rhdh[added a custom {product-short} application configuration].
+
+.Procedure
+. To allow {product-short} to authenticate with Microsoft Azure, create an OAuth Application in Microsoft Azure.
+
+.. Go to link:https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade[*Azure Portal > App registrations*], and create an *App Registration* for {product-short}.
+
+.. On your *App registration* overview page, add a new *Web platform configuration*, with the configuration:
```

Comment:
I don't see it either from the overview page. Maybe we should specify to navigate to Manage->Authentication from the Overview page:
![image](https://github.com/user-attachments/assets/543a523f-3ab5-4fab-b74e-079ddb7105d2)

Reply:
Seems it has changed since the initial docs were created.

I see something different (I have no running directory set up):

![Screenshot_2024-09-23_2](https://github.com/user-attachments/assets/3d82ed30-8f2b-498f-8dfa-74c8ff106b22)

Reply:
I stand corrected on my previous comment. The Web platform sections appears on App Registration creation (or when a Redirect URL s created for the first time), so the steps are actually correct. The `Manage->Authentication from the Overview page:` path from is valid when using an existing app. 
<img width="892" alt="image" src="https://github.com/user-attachments/assets/28e6cf00-95b9-462c-a163-0c3b086b56e9">
<img width="1491" alt="image" src="https://github.com/user-attachments/assets/7ec42c1a-1ebc-401e-b4e7-76dcde5579b2">